index.bs

```diff
@@ -44,12 +44,12 @@ Examples {#examples}
 
 <div class="example">
     <pre highlight="js">
-    let sensor = new DeviceOrientationSensor();
-    let matrix = new Float32Array(16);
+    const sensor = new DeviceOrientationSensor();
+    const mat4 = new Float32Array(16);
```

Comment:
using mat4 to indicate it is a 4x4 (ie 16) array. 

const is used when no reassignment occurs